distro.py

```diff
@@ -938,6 +958,7 @@ def _os_release_info(self):
         if os.path.isfile(self.os_release_file):
             with open(self.os_release_file) as release_file:
                 return self._parse_os_release_content(release_file)
+
```

Comment:
With my picky hat on: If this (leftover) addition of a blank line could be reverted, that would be perfect.